apps/docs/src/components/ex/Chat/style.css

```
@@ -2,22 +2,22 @@
   .b--chat {
     --cbox-bgPct: 8%;
     --keycolor: var(--gray);
-    gap: 0 calc(var(--_deco-size) + 1px);
+    gap: 0 calc(var(--deco-size) + 1px);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)